src/validateRegisterForm.test.js

```diff
@@ -27,4 +27,167 @@ describe(`Function 'validateRegisterForm':`, () => {
   });
 
   // write more tests here
+
+  it(`should return error for valid password and email with Cyrillic`, () => {
+    const isValid = validateRegisterForm('тест@mail.com', 'P@ssword1!');
+
+    expect(isValid.code).toBe(422);
+    expect(isValid.message).toBe('Email is invalid.');
+  });
+
+  it(`should return success message for valid password
+  and email with digits`, () => {
+    const isValid = validateRegisterForm('test123@mail.com', 'P@ssword1!');
+
+    expect(isValid.code).toBe(200);
+    expect(isValid.message).toBe('Email and password are valid.');
+  });
+
+  it(`should return success message for valid password and Character \`.\` 
```

Comment:
якщо ти хочеш використовувати лапки в лапках - просто використовуй інші лапки замість екранування 
```suggestion
  it(`should return success message for valid password and Character '.' 
```

Reply:
також занадто довга назва - треба скоротити
should return success response message for valid password and email with dots in personal_info